CRVResponse/src/CrvPhotonGenerator_module.cc

```diff
@@ -202,18 +210,31 @@ namespace mu2e
         {
            tableLoaded=true;
            _makeCrvPhotons.emplace_back(_makeCrvPhotons[j]);
-           std::cout<<"CRV sector "<<i<<" ("<<_CRVSectors[i]<<") uses "<<_makeCrvPhotons.back()->GetFileName()<<std::endl;
+           if(_debug>0) std::cout<<"CRV sector "<<i<<" ("<<_CRVSectors[i]<<") uses "<<_makeCrvPhotons.back()->GetFileName()<<std::endl;
            break;
         }
       }
       if(tableLoaded) continue;
 
       _makeCrvPhotons.emplace_back(boost::shared_ptr<mu2eCrv::MakeCrvPhotons>(new mu2eCrv::MakeCrvPhotons(_randFlat, _randGaussQ, _randPoissonQ)));
       boost::shared_ptr<mu2eCrv::MakeCrvPhotons> &photonMaker=_makeCrvPhotons.back();
-      photonMaker->LoadLookupTable(_resolveFullPath(_lookupTableFileNames[i]));
+      std::string filespec = _resolveFullPath(_lookupTableFileNames[i]);
+      filedirs.insert( std::filesystem::path(filespec).parent_path() );
+      photonMaker->LoadLookupTable(filespec,_debug);
       photonMaker->SetScintillationYield(_scintillationYields[i]);
-      std::cout<<"CRV sector "<<i<<" ("<<_CRVSectors[i]<<") uses "<<_makeCrvPhotons.back()->GetFileName()<<" with scintillation yield of "<<_scintillationYields[i]<<" photons/MeV"<<std::endl;
+      if(_debug>0) std::cout<<"CRV sector "<<i<<" ("<<_CRVSectors[i]<<") uses "<<_makeCrvPhotons.back()->GetFileName()<<" with scintillation yield of "<<_scintillationYields[i]<<" photons/MeV"<<std::endl;
+    }
+
+    std::cout << "CRV light files:";
+    for(auto const& dir : filedirs ){
+      std::cout << " " << dir;
+    }
+    std::cout << std::endl;
+    size_t hash = 0;
+    for(auto const& mcp: _makeCrvPhotons) {
+      boost::hash_combine<std::string>(hash,mcp->GetFileName());
     }
+    std::cout << "CRV light sectors: " << _makeCrvPhotons.size() << " hash:" << hash <<std::endl;
```

Comment:
The printout between lines 225 an 235 and on line 237 is not protected by a debug level. I am checking that this is intended.

Reply:
Yes, intended.  Recall when we discussed the print for the geometry and conditions files.  We decided that they should not be able to be turned off, since we wanted to force the logging of the critical inputs, including the hash.  This line follows that pattern.